tests/generic_views/test_detail.py

```diff
@@ -137,6 +140,34 @@ def test_duplicated_context_object_name(self):
         self.assertNotIn('author', res.context)
         self.assertTemplateUsed(res, 'generic_views/author_detail.html')
 
+    def test_deferred_queryset_template_name(self):
+        class FormContext(SingleObjectTemplateResponseMixin):
```

Comment:
If you are talking about SingleObjectTemplateResponseMixin, then it's the class responsible for generating template names from object meta attributes, which I also patched. It is needed in the test.